src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,53 @@
-import React from 'react';
+import * as React from 'react';
 import './CurrentUser.scss';
+import { getUsers } from '../../api/api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  selectedUserId: number,
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+type State = {
+  user: User | '',
+};
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    user: '',
+  };
+
+  async componentDidMount() {
+    const newUser = await getUsers(this.props.selectedUserId);
+
+    // eslint-disable-next-line react/no-did-update-set-state
+    this.setState({ user: newUser });
```

Comment:
make a separate function and call it here. You use the same code here and in componentDidUpdate